### ISSUE TITLE
How to run vtn_demo.py for AR

### ISSUE THREAD

**Issue by lianuo** [open-edge-platform/geti#80]
I  notice the demo file vtn_demo in 

https://github.com/opencv/openvino_training_extensions/tree/develop/pytorch_toolkit/action_recognition

where I can get the pretrained model for this program ?
where can i find the files correspond to "--checkpoint" and "--labels" 

Thanks

**Comment by snosov1** [comment]
Does the doc answer your questions? https://github.com/opencv/openvino_training_extensions/tree/develop/pytorch_toolkit/action_recognition#models

cc @vadimadr

**Comment by lianuo** [comment]
> Does the doc answer your questions? https://github.com/opencv/openvino_training_extensions/tree/develop/pytorch_toolkit/action_recognition#models
> 
> cc @vadimadr

Thank you ,But that doc does not mentioned the vtn_demo.py

**Comment by lianuo** [comment]
I have run the demo.
the  checkpoint is set to be path of   resnet_34_vtn_rgb_kinetics.pth
the "kinetics400_label.txt" file could generate by use this [file]([url](url))
thanks

**Comment by samanthawyf** [comment]
@lianuo hi, I want to run the demo. Could you share me the "kinetics400_label.txt" file? I don't have the Kinetics dataset yet and want to try the demo first.

**Comment by gmt710** [comment]
If anyone want to use vtn_demo or jpg's version, you can refer to [my chinese blog](https://blog.csdn.net/qq_35975447/article/details/112261368).
